vendor/engines/sanger_sequencing/spec/services/sanger_sequencing/sample_result_file_saver_spec.rb

```diff
@@ -50,7 +50,7 @@
 
     it "does not save" do
       expect(saver.save).to be(false)
-      expect(saver.errors[:name]).to include("Filename already exists for this order")
+      expect(saver.errors.messages_for(:name).flatten).to include("Filename already exists for this order")
```

Comment:
Can you try something like line 37 above?
```suggestion
      expect(saver.errors).to be_added(:name, "Filename already exists for this order")
```